openbb_terminal/stocks/fundamental_analysis/fa_controller.py

```diff
@@ -128,41 +128,41 @@ def print_help(self):
         mt.add_param("_ticker", self.ticker.upper())
         mt.add_raw("\n")
         mt.add_info("_company_overview")
-        mt.add_cmd("mktcap")
-        mt.add_cmd("overview")
-        mt.add_cmd("divs", not self.suffix)
-        mt.add_cmd("splits", not self.suffix)
-        mt.add_cmd("rating")
-        mt.add_cmd("rot")
-        mt.add_cmd("score")
-        mt.add_cmd("warnings")
+        mt.add_cmd("mktcap", not self.stock.empty)
+        mt.add_cmd("overview", not self.stock.empty)
+        mt.add_cmd("divs", not self.suffix and not self.stock.empty)
+        mt.add_cmd("splits", not self.suffix and not self.stock.empty)
+        mt.add_cmd("rating", not self.stock.empty)
+        mt.add_cmd("rot", not self.stock.empty)
+        mt.add_cmd("score", not self.stock.empty)
+        mt.add_cmd("warnings", not self.stock.empty)
         mt.add_raw("\n")
         mt.add_info("_management_shareholders")
-        mt.add_cmd("mgmt")
-        mt.add_cmd("shrs", not self.suffix)
-        mt.add_cmd("supplier")
-        mt.add_cmd("customer")
+        mt.add_cmd("mgmt", not self.stock.empty)
+        mt.add_cmd("shrs", not self.suffix and not self.stock.empty)
+        mt.add_cmd("supplier", not self.stock.empty)
+        mt.add_cmd("customer", not self.stock.empty)
         mt.add_raw("\n")
         mt.add_info("_financial_statements")
-        mt.add_cmd("income")
-        mt.add_cmd("balance")
-        mt.add_cmd("cash")
-        mt.add_cmd("growth")
-        mt.add_cmd("metrics")
-        mt.add_cmd("ratios")
-        mt.add_cmd("dupont")
-        mt.add_cmd("fraud")
-        mt.add_cmd("sec")
-        mt.add_cmd("analysis")
+        mt.add_cmd("income", not self.stock.empty)
+        mt.add_cmd("balance", not self.stock.empty)
+        mt.add_cmd("cash", not self.stock.empty)
+        mt.add_cmd("growth", not self.stock.empty)
+        mt.add_cmd("metrics", not self.stock.empty)
+        mt.add_cmd("ratios", not self.stock.empty)
+        mt.add_cmd("dupont", not self.stock.empty)
+        mt.add_cmd("fraud", not self.stock.empty)
+        mt.add_cmd("sec", not self.stock.empty)
+        mt.add_cmd("analysis", not self.stock.empty)
```

Comment:
these are all intentionally not dimmed because they can all be accessed using a ticker arg (income -t aapl).  Having them dimmed implied that something needs to be loaded first (which is why they are not dimmed in the ta menu)